src/transport_none.h

```diff
@@ -78,74 +78,74 @@ shmem_transport_fini(void)
 
 static inline
 int
-shmem_transport_quiet(void)
+shmem_transport_quiet(shmem_ctx_t ctx)
```

Comment:
You might want the type to be ```shmem_transport_ctx_t``` here to avoid casting all over the place in the implementation.  In the internal routines, you can cast the ```shmem_ctx_t ``` to ```shmem_transport_ctx_t *```.